src/lib/build/bundle.ts

```diff
@@ -76,9 +76,9 @@ export async function traceFiles(opts: Pick<BundleOptions, 'base' | 'plugins' |
     ignore: ['node_modules/prettier/index.js', 'node_modules/@prisma/client/scripts/**/*'],
     readFile(fsPath: string): Buffer | string | null {
       const relPath = Path.relative(opts.base, fsPath)
-
       const cached = sourceCache.get(relPath)
-      if (cached) return cached.toString()
+
+      if (cached) return cached.source
```

Comment:
That was the fix. I frankly don't know how it's been working til now 🤦‍♂️... I suspect the cache was never hit